sklearn/neural_network/multilayer_perceptron.py

```diff
@@ -1036,22 +1037,45 @@ def predict_log_proba(self, X):
         y_prob = self.predict_proba(X)
         return np.log(y_prob, out=y_prob)
 
-    def predict_proba(self, X):
+    def predict_proba(self, X, max_row_chunk=None):
         """Probability estimates.
 
         Parameters
         ----------
         X : {array-like, sparse matrix}, shape (n_samples, n_features)
             The input data.
 
+        max_row_chunk: integer
+            The number of rows to use at a single predict call. If None optimal 
+            row number fit that fits the working_memory is used.
```

Comment:
I don't think this needs an extra parameter, better to always use `working_memory`.

Reply:
👍